hi-ml-cpath/src/health_cpath/datasets/base_dataset.py

```diff
@@ -15,44 +15,41 @@
 from health_cpath.utils.naming import SlideKey
 
 
+TRAIN_SPLIT_LABEL = "train"  # Value used to indicate the training split in `SPLIT_COLUMN`
+VAL_SPLIT_LABEL = "val"  # Value used to indicate the validation split in `SPLIT_COLUMN`
+TEST_SPLIT_LABEL = "test"  # Value used to indicate the test split in `SPLIT_COLUMN`
```

Comment:
I moved it to avoid duplications in the code base.

Especially for testing where we were redefining these labels in MockTilesDataset which makes it hard to keep the mock dataset in sync with the base code.

So far, all datasets use "train"/"val"/"test" which is the standard afaik. These global variables should be used as default behaviour.

Maybe renaming them to `DEFAULT_TRAIN/TEST/VAL_SPLIT_LABEL` will make it clearer? 

I can also revert this change if you think that it reduces splitting flexibility and that it will affect your work on the test split.

Reply:
Makes sense. Renaming to `DEFAULT_*` sounds like a good compromise.